stack/documentation-system-overview.rst

```diff
@@ -0,0 +1,179 @@
+The primary examples are the *Modules* and *Packages* sections of the `pipelines_lsst_io`_ homepage that link to :doc:`module homepages <module-homepage-topic-type>` and :doc:`package homepages <package-homepage-topic-type>`, respectively.
+These sections are automatically populated by ``module-toctree`` and ``package-toctree``, which are specializations of Sphinx_\ ’s own `toctree`_ directive.
+In other words, the *Modules* and *Packages* sections are where package documentation content is added to the main content hierarchy of the `pipelines_lsst_io`_ Sphinx project.
+
+In addition to the *Modules* and *Packages* sections, the :ref:`plan <stack-docs-system-further-reading>` for the `pipelines_lsst_io`_ repository is to also have sections called *Processing* and *Frameworks*. 
+The *Processing* section will collect Task documentation and tutorials around specific data processing contexts (single-frame processing, for example).
+The *Frameworks* section will gather content from modules and packages around different API themes, called *frameworks,* to help make the package-based codebase easier to rationalize and navigate.
+For example, one framework would be the “Task framework,” consisting of APIs from `pipe_base`_, `pipe_supertask`_, and `pex_config`_.
+You can think of the *Processing* and *Frameworks* sections as *content playlists* that organize information across multiple packages along topical lines, outside the strict *Modules* and *Packages* hierarchy.
+
+.. _stack-docs-system-packages:
+
+Package documentation
+=====================
+
+Packages include their own documentation content in their :doc:`doc/ directories <layout-of-doc-directory>`.
+By maintaining documentation in the same repository as the code content, it is easier to keep documentation up-to-date with the implementation.
+
+Documentation for packages is split into type types of directories:
+
+-  One directory that describes the Git repository (package) itself (the :ref:`package documentation directory <docdir-package-doc-directory>`).
+-  The second type are :ref:`module documentation directories <docdir-module-doc-directories>` that describe the top-level packages provided by packages.
+
+Content within a package’s :doc:`doc/ directory <layout-of-doc-directory>` is written in :doc:`reStructuredText </restructuredtext/style>`.
+Each page conforms to a *topic type* (a template, essentially) that defines the style and structure of the content (see :doc:`package-documentation-topic-types`).
+By maintaining consistent information structures, documentation content is both easier to write and easier for readers to use and navigate.
+
+.. _stack-docs-system-build-overview:
+
+Overview of the documentation build process
+===========================================
+
+This section is a high-level overview of the tools and processes for building Stack documentation.
```

Comment:
Is this section necessary for people to know?  It seems to me that it might be better as a separate page.